assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -23,44 +23,51 @@ const DATE_RANGE = [{
     name: '1 year',
     period: 'P365D'
 }, {
-    name: 'Custom',
+    name: 'Custom dates/days',
     period: 'custom'
 }];
 const CUSTOM_TIMEFRAME_RADIO_BUTTON_DETAILS = [
+    {
+        id: 'custom-input-calendar-days',
+        value: 'calendar',
+        text: 'Calendar dates'
+    },
     {
         id: 'custom-input-days-before-today',
         value: 'days',
         text: 'Days before today'
-    },
-    {
-        id: 'custom-input-calendar-days',
-        value: 'calendar',
-        text: 'Calendar days'
     }
 ];
 
+// The standard 'selectedDateRange' options -- the options that are not a custom period, such as P129D or P3D
+const STANDARD_INITIAL_VALUES = ['custom', 'P7D', 'P30D', 'P1Y'];
+
 const isCustomPeriod = function(dateRange) {
     return dateRange !== 'custom' && !DATE_RANGE.find(range => range.period === dateRange);
 };
 const isCustomDateRange = function(dateRange) {
     return dateRange === 'custom';
 };
-const showCustomContainer = function(dateRange) {
+
+/*
+* Helper function that
```

Comment:
Might want to finish the sentence.

Reply:
Ugh, yes, I was going to write this documentation, and then I realized I didn't have the right words to describe what was happening in this really simple function, so I paused and subsequently forgot about it. I added a comment, to it now, but it seems like a lot of words for a simple thing, which may mean there is a better way to organize how these selection menus work. And since, we while probably change how is part of the application works in the near future it may be a good thing to think about then.